### PR TITLE
Analytics: Fix custom date range calendar styling in WP 6.1

### LINKED ISSUE #35645: Analytics: Custom date range calendar styling is broken in WordPress 6.1
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The custom date range calendar's styling on Analytics pages is broken in WordPress 6.1.

### Expected behavior

The custom date range calendar is styled properly.

### Actual behavior

The custom date range calendar's styling is broken.

image

### Steps to reproduce

1. Go to any Analytics page.
2. Click on the "Date range" and click "Custom".
3. Observe that the calendar styling is broken (see above).

### WordPress Environment

WordPress 6.1

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by mattsherman**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR includes the `react-dates` CSS that was previously included via WordPress. It is no longer included by WordPress 6.1+.

This CSS is required to properly style the calendar in the Analytics custom date range dropdown.

Screenshot 2022-11-20 at 10 12 15

Closes #35645.

- [ ] This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).

### How to test the changes in this Pull Request:

1. Go to any Analytics page.
2. Click on the "Date range" and click "Custom".
3. Verify that the calendar styling is fixed (see above).

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [x] Have you written new tests for your changes, as applicable?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm --filter=<project> changelog add`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: packages/js/components/src/calendar/style.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import "./react-dates.scss";
```
Comment (louwie17) [inline packages/js/components/src/calendar/style.scss]: how come you copy/pasted the entire css file of `react-dates.scss` in our repo, we should be able to import it from the installed package.
I tried it locally, it does look like our `css-loader` is a bit broken, so you did have to import it like this:
`@import '../node_modules/react-dates/lib/css/_datepicker.css';`

**Review by louwie17 — DISMISSED** [review]
Just left a suggested change to import it from the installed package instead.

File: packages/js/components/src/calendar/style.scss (hunk 1 above)
Comment (mattsherman) [reply packages/js/components/src/calendar/style.scss]: Honestly, I copied the approach `@wordpress/components` had used. It never dawned on me to just import from the installed package. I will look into that. Though, I do wonder if there are any differences... perhaps even "just" the RTL handling.

Comment (mattsherman) [reply packages/js/components/src/calendar/style.scss]: Addressed this in 930cb5c

**Comment by mattsherman** [comment]
Ready for a re-review!

**Review by joelclimbsthings — DISMISSED** [review]
Thanks for the fix @mattsherman . Tested in Firefox/Chrome and compared to trunk; BIG improvement!  Interesting work-around with the imports as well. 🚢 :shipit: 

Edit: I also took the liberty of reruning the e2e tests which had failed. Looked like an unrelated timeout.

**Comment by octaedro** [comment]
Hey @joelclimbsthings,

I just rebased and pushed this PR. Could you take another look at this PR?

**Review by octaedro — APPROVED** [review]
LGTM! 🚀

**Comment by kaushikasomaiya** [comment]
@mattsherman Wondering if this temporary fix needs to be removed since we've imported react dates, on wccom repo: https://github.com/Automattic/woocommerce.com/pull/15003/

**Pull request merged**